Homework 3 completed

GPMila79 opened the pull request:
Added four files with datas and solutions

GPMila79 committed:
Added datas for first task

GPMila79 committed:
Changed sign in the last example

GPMila79 committed:
Added datas for second task

GPMila79 committed:
Added datas for third task

GPMila79 committed:
Added datas for fourth task

eugene-okulik reviewed (changes requested):
Всё задание должно быть выполнено в питоновских файлах. У них расширение `.py`

GPMila79 committed:
Changed page resolution

GPMila79 committed:
Fixed erros from lint

GPMila79 committed:
Delete blank lines in file 1 and 2

GPMila79 commented:
1. Changed page resolution in files (txt in py)
2. Fixed erros from lint (blank line at end of file, no newline at end of file, whitespace)
3. Fixed erros from lint (blank line at end of file 1 and file 2)
